lib/common.ts

```diff
@@ -0,0 +1,2 @@
+//Common functions for test suite
+export const sleep = (ms: number) => new Promise((r) => setTimeout(r, ms));
```

Comment:
How is this not creating merge conflicts? 😄 That file already exists on `main` branch and was supposed to be brought into this branch just with rebase, or Am I wrong? 😄

Reply:
It was brought by rebase to this branch from main and there are no conflicts otherwise they would show up here

Reply:
Weird, thought that commit hash would match and it would not show up in `Files changed` if it was rebased.

Reply:
I have not reviewed the entire PR but I would like to chip in on this line.

I **strongly discourage** you to have `sleep` as a "common" functionality, it should **NOT** be used with the only exception of *temporary workarounds* for actual bugs.

Making it a common functionality somewhat encourages the usage hiding deeper problems in the stack and less solid implementations in this very repository that should be using better selectors and leverage Playwright `autowait` functionalities.

cc. @Frawless for an additional opinion on this.

Reply:
I think a PR disabling the sleep functionality has already been merged

Reply:
More info about why it was introduced is here https://github.com/redhat-developer/consoledot-e2e/pull/60#issuecomment-1371140174